meshmode/interop/firedrake/connection.py

```diff
@@ -39,9 +39,8 @@
 from meshmode.mesh.processing import get_simplex_element_flip_matrix
 
 from meshmode.discretization.poly_element import (
-    PolynomialWarpAndBlendGroupFactory,
-    PolynomialRecursiveNodesGroupFactory,
-    ElementGroupFactory)
+        default_simplex_group_factory,
+        ElementGroupFactory)
```

Comment:
```suggestion
    default_simplex_group_factory,
    ElementGroupFactory)
```
? Everyone else seems to be indented like this.

Reply:
Gone in 8a822fe.